src/lib/components/primitives/label/label.stories.js

```diff
@@ -15,3 +17,17 @@ export const Base = {
     label: 'Username',
   },
 };
+
+
+const Template = (args) => ({
+  Component: Label,
+  props: args
+});
+export const Interaction = Template.bind({});
+Interaction.args = { ...Base.args };
+Interaction.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const text = canvas.getByText('Input Label')
+  expect(text).toBeInTheDocument();
+}
```

Comment:
I do feel this is unnecessary. If we have a lot of these, it can slow down our CI and just generally cause delays, timeouts or other issues for very little or no actual benefit. If something is this simple, we can just not test it. I think it's okay for some things to not be tested.

Reply:
For me, it renders `Input Label` on the storybook.

Reply:
Yeah, that's because we don't pass through the text intended to be in the label in the `label.story.ts` component, which is incorrect.